docs/source/apps/max_sharpe_ratio_optimization.ipynb

```diff
@@ -0,0 +1,1042 @@
+{
```

Comment:
<p>Nit: "Given that understanding we will go through an example Sharpe ratio optimization. As an example dataset, we can use stocks <strong>from companies in the S\\&amp;P 500 (available on Wikipedia) for our portfolio</strong>:"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SupertechLabs/client-superstaq/pull/417/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>